backend/src/main/java/ca/bc/gov/hlth/accounttransfer/controller/AccountsController.java

```diff
@@ -111,6 +113,13 @@ public ResponseEntity<AccountTransferResponse> transferAccount(@Valid @RequestBo
 							ldapResponse.getMspDirectRole().toUpperCase(), MSPDIRECT));
 			return ResponseEntity.ok(response);
 		}
+		List<String> orgDetails = loadOrgDetails(user);
```

Comment:
We can remove the oldLdapId check above this (and any code it calls) as this condition is handled in the UI. Having backup code in the API isn't necessary as this is an edge case and not a security issue.